nwb_conversion_tools/datainterfaces/behavior/movie/movie_utils.py

```diff
@@ -13,9 +14,112 @@
 PathType = Union[str, Path]
 
 
+class VideoCaptureContext:
+    """Retrieving video metadata and frames using a context manager"""
+
+    def __init__(self, file_path: FilePathType, stub=False):
+        self.vc = cv2.VideoCapture(file_path)
```

Comment:
```suggestion
        self.vc = cv2.VideoCapture(filename=file_path)
```

I always strongly suggest using keywords when possible; we've had errors from the old days that were caused by non-transparent changes to positional orders and they were a complete pain to debug.

Reply:
Its a very core functionality of `VideoCapture`, its hard to imagine any other arg taking precedence over file path as first arg. Did you come across such opencv api changes?